libbeat/cmd/version.go

```diff
@@ -22,8 +22,15 @@ func genVersionCmd(name, beatVersion string) *cobra.Command {
 					return fmt.Errorf("error initializing beat: %s", err)
 				}
 
-				fmt.Printf("%s version %s (%s), libbeat %s\n",
-					beat.Info.Beat, beat.Info.Version, runtime.GOARCH, version.GetDefaultVersion())
+				var buildTime string
```

Comment:
Consider initializing `buildTime := "unknown"`. I think it could make this a bit more concise.